### PR TITLE
Adjust for deprecation of `--monitor`

### PR DESCRIPTION
**Pull request by dveeden**
### What problem does this PR solve? 

Replace `--monitor` in various places

### Check List 

Tests 

 - No code

Release notes:
```release-note
NONE
```

### CODE REVIEW

File: doc/user/playground.md [hunk 1]
```diff
@@ -22,23 +22,48 @@ The command line arguments for playground state:
...
+  completion  generate the autocompletion script for the specified shell
+  display
+  help        Help about any command
+  scale-in
+  scale-out
 
 Flags:
-      --db int                   TiDB instance number (default 1)
+      --db int                   TiDB instance number
+      --db.Host host             Playground TiDB host. If not provided, TiDB will still use host flag as its host
```
Comment (AstroProfundis) [inline doc/user/playground.md:36]: This should be in lower case, it was a bug fixed in #1635

Comment (dveeden) [reply doc/user/playground.md:36]:
```suggestion
      --db.host host             Playground TiDB host. If not provided, TiDB will still use host flag as its host
```

File: doc/user/playground.md [hunk 2]
```diff
@@ -22,23 +22,48 @@ The command line arguments for playground state:
...
+  display
+  help        Help about any command
+  scale-in
+  scale-out
 
 Flags:
-      --db int                   TiDB instance number (default 1)
+      --db int                   TiDB instance number
+      --db.host host             Playground TiDB host. If not provided, TiDB will still use host flag as its host
+      --db.Port int              Playground TiDB port. If not provided, TiDB will use 4000 as its port
```
Comment (dveeden) [inline doc/user/playground.md:37]:
```suggestion
      --db.port int              Playground TiDB port. If not provided, TiDB will use 4000 as its port
```

File: doc/user/playground.md (hunk 1 above)
Comment (dveeden) [reply doc/user/playground.md]: This isn't yet fixed in v1.7.0 / ce8eb0a645cc3ead96a44d67b1ecd5034d112cf0

It is fixed in the master branch ( 9edf64ea0c1917a8aab2c355448471aa5e1419ee )

**Review by AstroProfundis — APPROVED** [review]

**Pull request merged**